src/withStyles.jsx

```diff
@@ -56,15 +75,7 @@ export function withStyles(
           ThemedStyleSheet.flush();
         }
 
-        const addedProps = {
-          [themePropName]: ThemedStyleSheet.get(themeName),
```

Comment:
Is ThemedStylesheet.get guaranteed to be idempotent based on the themeName?

If not, you may need to call it unconditionally in cWRP.